pom.xml

```diff
@@ -432,6 +460,18 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.8.1</version>
                     <configuration>
+                        <annotationProcessorPaths>
+                            <path>
+                                <groupId>com.google.auto.service</groupId>
+                                <artifactId>auto-service</artifactId>
+                                <version>${version.auto-service}</version>
+                            </path>
```

Comment:
Auto service is _currently_ not used by any of our OSS libraries (also not `jooq-generator-maven-plugin`), but I expect it will at some point, and adding it now prevents me from instead having to explicitly declare a Guava dependency to resolve an incompatibility between Error Prone and Immutables.